File: sample/src/main/java/vn/tiki/sample/collectionview/LoadingViewHolder.java
package vn.tiki.sample.collectionview;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import vn.tiki.noadapter2.AbsViewHolder;
import vn.tiki.sample.R;

public class LoadingViewHolder extends AbsViewHolder {

  private LoadingViewHolder(View view) {
    super(view);
  }

  public static LoadingViewHolder create(ViewGroup parent) {
    final View view = LayoutInflater.from(parent.getContext()).inflate(
        R.layout.clv_item_loading,
        parent,
        false);
    return new LoadingViewHolder(view);
  }
}

File: sample/src/main/java/vn/tiki/sample/collectionview/LoadingViewHolder.kt
package vn.tiki.sample.collectionview

import android.view.View
import android.view.ViewGroup
import vn.tiki.noadapter2.AbsViewHolder
import vn.tiki.sample.R
import vn.tiki.sample.util.inflate

class LoadingViewHolder private constructor(view: View) : AbsViewHolder(view) {
  companion object {
    fun create(parent: ViewGroup): LoadingViewHolder {
      val view = parent.inflate(R.layout.clv_item_loading)
      return LoadingViewHolder(view)
    }
  }
}
